src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/LoadStatus.java

```diff
@@ -94,29 +91,63 @@ public void writeLoadStatusFinished(long sampleId) {
-        // towards the CreateStreamWriter quota
-        try (JsonStreamWriter writer =
-                     JsonStreamWriter.newBuilder(loadStatusTable.toString(), getLoadStatusTableSchema()).build()) {
-
-            // Create a JSON object that is compatible with the table schema.
-            JSONArray jsonArr = new JSONArray();
-            JSONObject jsonObject = new JSONObject();
-            jsonObject.put("sample_id", sampleId);
-            jsonObject.put("status", status.toString());
-            jsonObject.put("event_timestamp", System.currentTimeMillis() * 1000L); // google wants this in microseconds since epoch...
-            jsonArr.put(jsonObject);
-
-            ApiFuture<AppendRowsResponse> future = writer.append(jsonArr);
-            AppendRowsResponse response = future.get();
-
-            logger.info("Load status " + status + " appended successfully");
-        } catch (ExecutionException | InterruptedException | Descriptors.DescriptorValidationException | IOException e) {
-            // If the wrapped exception is a StatusRuntimeException, check the state of the operation.
-            // If the state is INTERNAL, CANCELLED, or ABORTED, you can retry. For more information, see:
-            // https://grpc.github.io/grpc-java/javadoc/io/grpc/StatusRuntimeException.html
-            throw new GATKException(e.getMessage());
+        final ExponentialBackOff backoff = new ExponentialBackOff.Builder().
+                setInitialIntervalMillis(2000).
+                setMaxIntervalMillis(30000).
+                setMultiplier(2).
+                setRandomizationFactor(0.5).
+                build();
+        int retryCount = 0;
+        int maxRetries = 3;
+
+        while (true) {
+            // This uses the _default stream since it (a) commits immediately and (b) doesn't count
+            // towards the CreateStreamWriter quota
+            try (JsonStreamWriter writer =
+                         JsonStreamWriter.newBuilder(loadStatusTable.toString(), getLoadStatusTableSchema()).build()) {
+
+                // Create a JSON object that is compatible with the table schema.
+                JSONArray jsonArr = new JSONArray();
+                JSONObject jsonObject = new JSONObject();
+                jsonObject.put("sample_id", sampleId);
+                jsonObject.put("status", status.toString());
+                jsonObject.put("event_timestamp", System.currentTimeMillis() * 1000L); // google wants this in microseconds since epoch...
+                jsonArr.put(jsonObject);
+
+                ApiFuture<AppendRowsResponse> future = writer.append(jsonArr);
+                future.get();
+
+                logger.info("Load status " + status + " appended successfully");
+                break;
+            } catch (Exception e) {
+                @SuppressWarnings("ThrowableNotThrown")
+                StatusRuntimeException se = handleCausalStatusRuntimeException(e);
```

Comment:
Nit: this method doesn't actually handle anything, maybe rename to `extract` instead of `handle` or something… but only if you're making another commit to this PR anyways